exporter/signalfx/signalfx.go

```diff
@@ -0,0 +1,293 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package signalfx contains a SignalFx exporter that supports exporting
+// OpenCensus views as SignalFx data points.
+package signalfx // import "opencensus-go-signalfx/exporter/signalfx"
+
+import (
+	"bytes"
+	"context"
+	"log"
+	"sync"
+	"time"
+
+	"errors"
+	"fmt"
+	"strings"
+	"unicode"
+
+	"github.com/signalfx/golib/datapoint"
+	"github.com/signalfx/golib/sfxclient"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+// Exporter exports stats to SignalFx
+type Exporter struct {
+	// Options used to register and log stats
+	opts Options
+	c    *collector
+}
+
+// Options contains options for configuring the exporter.
+type Options struct {
+	// Token contains the required token to send datapoints
+	// to SignalFx. It cannot be empty
+	Token string
+
+	// DatapointEndpoint contains the endpoint to send the datapoints
+	// to SignalFx.
```

Comment:
Document that the default can be found as `sfxclient.IngestEndpointV2`.

Reply:
done